codegens/http/lib/util.js

```diff
@@ -1,7 +1,7 @@
 let _ = require('./lodash'),
   path = require('path');
 
-const FORM_DATA_BOUNDARY = '----WebKitFormBoundary7MA4YWxkTrZu0gW',
+const FORM_DATA_BOUNDARY = '--WebKitFormBoundary7MA4YWxkTrZu0gW',
```

Comment:
@saksham-postman It seems we have a reason to keep `----` here. Chrome dev tools uses `----` as starting sequence for boundary.

I think we should keep using `----` instead of just `--` for better compatibility.

You can see this issue to see how the boundary can be seen in dev tools for reference. (https://github.com/postmanlabs/postman-app-support/issues/11084).